arches/app/media/js/views/components/widgets/text.js

```diff
@@ -180,6 +180,10 @@ define([
 
         });
 
+        if (self.currentDefaultText() === "") {
+            self.defaultValue("");
```

Comment:
@SDScandrettKint, @jacobtylerwalls - I was looking at this as it was causing me issues (of my own making) and in look at this I was wondering if line 184 should be:
` self.defaultValue(initialDefault);`
rather than setting the value to an empty string. I think (?) that the value and defaultValues should be the i18n object unless I've got that wrong... If my assumption is correct, not sure if this might bite us somewhere down the road...

Reply:
I was wondering along the same lines [here](https://github.com/archesproject/arches/pull/10642#issuecomment-1979332998), but I remember being less concerned about it when I saw other logic that watches the value and supplies the missing language keys (I think?)

This whole viewmodel is pretty involved, so I'm hoping we can kind of just make it to the Vue cutover without having to refactor it.